app/views/friendships/_friends_list.html.erb

```diff
@@ -0,0 +1,39 @@
+<% obj.each do |friendship| %>
+<tbody class = "friend-row">
+	<tr>
+		<td ><strong>Summoner Name: <%= friendship.friend.name.capitalize %></strong></td>
+		<% if @friends_list_code == 'pending' %> 
+		  <td ><strong> <%if friendship.initiator == false %>
+		                  Received: 
+		                <% else %>
+		                  Sent: 
+		                <% end %> 
+		    <%= "#{time_ago_in_words(friendship.created_at)}".capitalize %> ago</strong></td>
```

Comment:
You don't need string interpolation for all these method calls.  That just slows the interpreter down.
You only need interpolation if you want to combine a string with a method call.  For example:
`"Hello, my name is #{user.name_var}"`.  If you just want the value of my name, you would just call `user.name` with no interpolation.  Check all of the views you made in this pull request and remove any unnecessary string interpolation.